config/prod/kristen-bucket.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "kristen-bucket"
```

Comment:
bucket name containing `kristen-bucket` is ok?  wouldn't you rather this bucket's name be something related to the project?

Reply:
ah I wasn't sure if Kristen wanted that name or not. Do you want me to change the template name if I change the stack name?

Reply:
yes, stack_name and file name must match.